src/scripts/components/ToggleButton.jsx

```diff
@@ -5,7 +5,7 @@ import React, { Component } from 'react';
 class ToggleButton extends Component {
   handleClick = () => {
     if (this.props.onClick) {
-      this.props.onClick(this.props.variable.value, this.props.variable.id);
+      this.props.onClick(this.props.variable.variable, this.props.variable.id);
```

Comment:
Is this intended?

Reply:
it is. i was doing some goofiness before that i really didnt neeed to do.

Reply:
Can you elaborate on this?

Reply:
It was leftover from changes i was trying to make / test in rhinofront  by inserting the variable value (ie First Name) instead of the actual variable ({first_name}). I thought that by inserting the value I would be able to avoid having to transform the actual variable, but that ended up not being the case. Here i'm just changing it back to how its supposed to be

Reply:
I see. `variable.variable` is kinda weird. Ideally the names would be different to help differentiate them. If it was already like that, then I'm fine with it.